packages/svelte/package.json

```
@@ -58,7 +58,8 @@
       "default": "./src/runtime/store/index.js"
     },
     "./internal/disclose-version": {
-      "import": "./src/runtime/internal/disclose-version/index.js"
+      "import": "./src/runtime/internal/disclose-version/index.js",
+      "default": "./src/runtime/internal/disclose-version/index.js"
```

To align it with others this should suffice
```suggestion
      "default": "./src/runtime/internal/disclose-version/index.js"
```